src/Raven.Server/Documents/Handlers/Admin/Processors/Revisions/AbstractAdminRevisionsHandlerProcessorForDeleteRevisions.cs

```diff
@@ -15,24 +20,42 @@ protected AbstractAdminRevisionsHandlerProcessorForDeleteRevisions([NotNull] TRe
         {
         }
         
-        protected abstract ValueTask DeleteRevisionsAsync(TOperationContext context, string[] documentIds, bool includeForceCreated,
+        protected abstract Task<long> DeleteRevisionsAsync(DeleteRevisionsOperation.Parameters parameters,
             OperationCancelToken token);
 
         public override async ValueTask ExecuteAsync()
         {
-            bool includeForceCreated = RequestHandler.GetBoolValueQueryString("includeForceCreated", required: false) ?? false;
-
+            DeleteRevisionsOperation.Parameters parameters;
             using (ContextPool.AllocateOperationContext(out TOperationContext context))
             {
                 var json = await context.ReadForMemoryAsync(RequestHandler.RequestBodyStream(), "admin/revisions/delete");
-                var parameters = JsonDeserializationServer.Parameters.DeleteRevisionsParameters(json);
+                parameters = JsonDeserializationServer.Parameters.DeleteRevisionsParameters(json);
+            }
 
-                using (var token = RequestHandler.CreateHttpRequestBoundTimeLimitedOperationToken())
-                {
-                    await DeleteRevisionsAsync(context, parameters.DocumentIds, includeForceCreated, token);
-                }
+            parameters.Validate();
+            if (LoggingSource.AuditLog.IsInfoEnabled)
+            {
+                RequestHandler.LogAuditFor(RequestHandler.DatabaseName,
+                    "DELETE", $"Delete Revisions ,parameters: {JsonSerializer.Serialize(parameters)}");
```

Comment:
I don't know if we want to use the `JsonSerializer` from `System.Text.Json`, it is the first usage in this project..
let's simply use the `json` blittable instead.

Reply:
Done